src/components/MySearching/index.tsx

```diff
@@ -0,0 +1,10 @@
+interface  serachInterface {
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+export const MySearching: React.FC<serachInterface> = ({
+    onChange
+  }) => {
+    return(
+        <input onChange={onChange} type="text" placeholder="Search" className=" placeholder-white text-white pl-2 w-[548px] h-[45px] bg-[#62676e] border rounded-lg mb-[200px] mt-8"/>
```

Comment:
This component is just styles, I suggest passing all html attributes as props here, and throwing them inside input
by using {...props}